app/models/host_aspect_base.rb

```diff
@@ -0,0 +1,36 @@
+class HostAspectBase < ActiveRecord::Base
+  self.abstract_class = true
+
+  def self.inherited(subclass)
+    super
+    subclass.class_eval do
+      belongs_to_host
+    end
+  end
+
+  def info
```

Comment:
Comments on how to override these generic methods would be useful. Maybe something should be raised instead of returning a default, or are these ones meant to be used?